tests/test_issue_421_scope.py

```
@@ -5,12 +5,21 @@
 from mnemosyne.core.beam import BeamMemory
 
 
+def _use_empty_config(tmp_path, monkeypatch):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_use_empty_config`

Add return type annotation: `None`

(ANN202)